app/vmauth/auth_config_test.go

```diff
@@ -799,6 +799,36 @@ func TestBrokenBackend(t *testing.T) {
 	}
 }
 
+func TestDiscoverBackendIPsWithIPV6(t *testing.T) {
```

Comment:
It'd be great to use `f-tests` pattern for tests https://itnext.io/f-tests-as-a-replacement-for-table-driven-tests-in-go-8814a8b19e9e

Reply:
I've updated the tests to use that pattern. Makes the test clean and easy to understand